Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -356,6 +360,19 @@ private void OnInteractUsing(Entity<PlantHolderComponent> entity, ref InteractUs
         }
     }
 
+    private void OnSolutionAdded(Entity<PlantHolderComponent> entity, ref SolutionTransferSuccessEvent args)
+    {
+        var (uid, component) = entity;
+
+        //Only play splosh sound when using containers that would be dumped into the basin (as opposed to ie. sprayed on)
+        if (TryComp(args.From, out SpillableComponent? spill))
+        {
+            if(component.WateringSound != null)
+            {
+                _audio.PlayPvs(component.WateringSound, uid);
+            }
```

Comment:
```suggestion
            _audio.PlayPvs(component.WateringSound, uid);
```
you dont need to check for null, as all the audio events take in nullable soundspecifiers.